hotfixes/5.0.10.cfg

```diff
@@ -0,0 +1,13 @@
+[buildout]
+
+hotfix-eggs =
+    Products.PloneHotfix20171128
```

Comment:
According to https://pypi.org/project/Products.PloneHotfix20171128/ and the matrix at https://plone.org/security/hotfixes Plone version`5.0.10` should contain `Products.PloneHotfix20171128`, no?

Reply:
Yep, I double checked. The matrix at https://plone.org/security/hotfixes seems to be correct, and `Products.PloneHotfix20171128` was included in:
- 4.3.16
- 5.0.10
- 5.1.0.1
- 5.2a1

Sorry, didn't spot this in your original PR that updated the config :-/

Reply:
@lukasgraf thanks for spotting the problem!

We are going to fix that in https://github.com/jone/plone-hotfixes/pull/13, then update this pull request.

Reply:
@lukasgraf I have updated the hotfixes with the script.